src/nvim/version.c

```diff
@@ -36,7 +36,9 @@ char *Version = VIM_VERSION_SHORT;
 char *longVersion = NVIM_VERSION_LONG;
 char *longVersionWithDate = NVIM_VERSION_LONG " (compiled " __DATE__ " " __TIME__ ")";
 char *mediumVersion = NVIM_VERSION_MEDIUM;
+#ifdef NVIM_VERSION_HAS_COMMIT
```

Comment:
Okay, changed this around and put an `ifdef VERSION_HAS_COMMIT` in `versiondef.h.in`.